src/geom/cell_pyramid.C

```diff
@@ -46,7 +46,8 @@ const Real Pyramid::_master_points[14][3] =
     {0, -0.5, 0.5},
     {0.5, 0, 0.5},
     {0, 0.5, 0.5},
-    {-0.5, 0, 0.5}
+    {-0.5, 0, 0.5},
+    {0, 0, 0}
```

Comment:
Nice catch. I will cherry-pick this onto [branch_v1.6.2](https://github.com/libMesh/libmesh/tree/branch_v1.6.2) as well.

Reply:
Thanks!